java/bogdana_maksiuta/src/com/zoolatech/lecture5/tasks/_3/UniqueElements.java

```diff
@@ -0,0 +1,33 @@
+package com.zoolatech.lecture5.tasks._3;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+
+/**
+ * Create a method that accepts a list of words, and displays only unique one (order can be random).
+ *
+ * Bonus task: treat strings in a different case as the same one: “Apple” and “apple” should be treated
+ * as the same string (don’t use any methods that create a new string object -
+ * like toUpperCase or toLowerCase - you can work only with a given input).
+ *         a. Input: [“apple”, “orange”, “pear”, “apple”, “banana”, “orange”]
+ * Output: apple
+ * 			orange
+ * 			pear
+ * 			banana
+ */
+public class UniqueElements {
+
+    static void uniqueWords (List<String> words) {
+        Set<String> wordsSet = new TreeSet<>(String.CASE_INSENSITIVE_ORDER);
+        wordsSet.addAll(words);
+        System.out.println(wordsSet);
+    }
+
+    public static void main(String[] args) {
+        List<String> fruits = List.of("apple","orange", "pear", "apple", "banana", "oRange");
+        uniqueWords(fruits);
```

Comment:
prefer to define methods in their usage order: in a current case method `main` calls method `uniqueWords`, so it's better to define them in the same order: firstly `main`, then - `uniqueWords`